examples/move/function_keys/Move.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "function_keys"
+edition = "2024" # edition = "legacy" to use legacy (pre-2024) Move
+# license = ""           # e.g., "MIT", "GPL", "Apache 2.0"
+# authors = ["..."]      # e.g., ["Joe Smith (joesmith@noemail.com)", "John Snow (johnsnow@noemail.com)"]
+
+[dependencies]
+Iota = { local = "../../../crates/iota-framework/packages/iota-framework" }
+iotaccount = { local = "../../move/iotaccount" }
```

This should work as well
```suggestion
iotaccount = { local = "../iotaccount" }
```

---

Done